app/views/data_exports/_new_export_dialog.html.erb

```diff
@@ -5,34 +5,26 @@
         data: {
           turbo: false
         }, method: :post) do |form| %>
-    <div
-      data-controller="sessionstorage-amend-form"
-      data-sessionstorage-amend-form-target="field"
-      data-sessionstorage-amend-form-field-name-value="data_export[export_parameters[ids]][]"
-    />
     <div class="mb-4 font-normal text-slate-500 dark:text-slate-400">
-      <span
-        data-controller="data-export"
-        data-data-export-target="summary"
-        data-data-export-singular-value="<%= t(".summary.start.singular") %>"
-        data-data-export-plural-value="<%= t(".summary.start.plural") %>"
-        class="mb-4">
-        <%= t(".summary.start.zero") %>
-      </span>
-      <%= viral_pill(
-        text: t(".summary.processing"),
-        color: :gray
-      ) %>
-      <span>
-      <%= t(".summary.middle") %>
-      </span>
-      <%= viral_pill(
-        text: t(".summary.ready"),
-        color: :green
-      ) %>
-      <span>
-        <%= t(".summary.end") %>
-      </span>
+      <% if export_type == 'sample' %>
+        <div
+          data-controller="sessionstorage-amend-form"
+          data-sessionstorage-amend-form-target="field"
+          data-sessionstorage-amend-form-field-name-value="data_export[export_parameters[ids]][]"
+        />
+        <span
+          data-controller="data-export"
+          data-data-export-target="summary"
+          data-data-export-singular-value="<%= t(".summary.sample.singular", processing: t(".summary.processing"), ready: t(".summary.ready")) %>"
+          data-data-export-plural-value="<%= t(".summary.sample.plural", processing: t(".summary.processing"), ready: t(".summary.ready")) %>"
+          class="mb-4">
+          <%= t(".summary.sample.zero", processing: t(".summary.processing"), ready: t(".summary.ready")) %>
+        </span>
+      <% else %>
+        <span>
```

Comment:
**nitpick:** Where you have these `span` tags that have text in them, can you update them to be `<p class="text-base leading-relaxed text-slate-500 dark:text-slate-400">`

**Before**  
 

![image](https://github.com/phac-nml/irida-next/assets/11295750/e0048d59-2cd8-4907-b7b4-11cb2226e28e)

**After**  
 

![image](https://github.com/phac-nml/irida-next/assets/11295750/5bc4ee69-22e2-4aec-b112-92e3b4569937)

The `p` tag makes it more semantic, and the leading-relaxed just makes it slightly easier to read.  I know it is a small difference

Reply:
Thanks